src/LuaTransformer.ts

```diff
@@ -471,6 +473,39 @@ export class LuaTransformer {
         }
     }
 
+    public transformImportEqualsDeclaration(declaration: ts.ImportEqualsDeclaration): StatementVisitResult {
+        const name = this.transformIdentifier(declaration.name);
+        const expression = this.transformModuleReference(declaration.moduleReference);
+
+        return tstl.createVariableDeclarationStatement(name, expression, declaration);
+    }
+
+    public transformModuleReference(moduleReference: ts.ModuleReference): ExpressionVisitResult {
+        return ts.isExternalModuleReference(moduleReference)
+            ? this.transformExternalModuleReference(moduleReference)
+            : this.transformEntityName(moduleReference);
+    }
+
+    public transformExternalModuleReference(
+        externalModuleReference: ts.ExternalModuleReference
+    ): ExpressionVisitResult {
+        return this.transformExpression(externalModuleReference.expression);
+    }
+
+    public transformEntityName(entityName: ts.EntityName): ExpressionVisitResult {
+        return ts.isQualifiedName(entityName)
+            ? this.transformQualifiedName(entityName)
+            : this.transformIdentifier(entityName);
+    }
+
+    public transformQualifiedName(qualifiedName: ts.QualifiedName): ExpressionVisitResult {
+        const right = this.transformIdentifier(qualifiedName.right);
+        const rightLiteral = tstl.createStringLiteral(right.text, qualifiedName.right);
```

Comment:
> I feel like there has to be a better way to do transformers.ts:503, but I don't know how.

`getIdentifierText`?

Reply:
That would lose the renaming logic that's in transformIdentifier.